uuid/mod.ts

```diff
@@ -2,17 +2,26 @@
 // This module is browser compatible.
 
 /**
- * Generators and validators for UUIDs for versions v1, v3, v4 and v5.
+ * Generators and validators for [RFC4122](https://www.rfc-editor.org/rfc/rfc4122.html)
+ * UUIDs for versions v1, v3, v4 and v5.
  *
- * Use {@linkcode crypto.randomUUID} for v4 generating v4 UUIDs.
+ * Use the built-in [`crypto.randomUUID`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/randomUUID)
+ * function instead of this package, if you only need to generate v4 UUIDs.
  *
- * Based on {@linkcode https://www.npmjs.com/package/uuid | npm:uuid}, which is
- * based on {@link https://www.rfc-editor.org/rfc/rfc4122.html | RFC 4122}.
- *
- * Support for RFC4122 version 1, 3, 4, and 5 UUIDs
+ * Based on {@linkcode https://www.npmjs.com/package/uuid | npm:uuid}.
  *
  * This module is browser compatible.
  *
+ * @example
```

Comment:
Nit: So the code snippet title isn't "Example 1" on JSR.
```suggestion
 * @example Usage
```